File: services/python/sanic/src/routes/handlers/internal.py
import structlog
from sanic import Request
from sanic.response import JSONResponse

from routes.errors import AppErrors, send_error


async def internal_handler(request: Request, exception: Exception) -> JSONResponse:
    structlog.get_logger().error(
        "Internal server error",
        method=request.method,
        path=request.path,
        error=repr(exception),
    )

    return send_error(AppErrors.Internal)

File: services/go/fiber/routes/handlers/internal.go
package handlers

import (
	"fiber/routes"
	"log/slog"

	"github.com/gofiber/fiber/v3"
)

func InternalHandler(c fiber.Ctx, err any) error {
	slog.Error("Internal server error", "method", c.Method(), "path", c.Path(), "error", err)

	return routes.AppErrors.Internal.Error(c)
}
